python/lsst/sattle/app/main.py

```diff
@@ -36,8 +44,9 @@ def read_tle_from_url(url, tle_source):
 
         response = requests.get(test_url, params=params, timeout=70)
 
+        # This needs to be swapped so the start and end brackets the start time
+        # By a specific set of hours so we don't have crazy offsets in epoch.
```

Comment:
Again, comment is not clear, did you already do whatever the swap is?

Reply:
Will be worked on in a separate ticket.

Reply:
Please mention the ticket number in the comment so it doesn't fall off your radar